README.md

```diff
@@ -80,6 +80,7 @@ Creates a new GoodInflux object where:
   `my/awesome/service/ops`
   - `[prefixDelimiter]` - Used to delimit measurement prefix arrays defined in *prefix* above. Defaults to `/`.
   - `[customLogFormatters]` - This options allows you to report specific log events based on tag (Note the first matching tag in `event.tags` will be used, it's better to have specific tags in practice). Ordinarily, log events are sent as a string. However, you may want to extract a custom data field and send it to Influx in a more searchable format. For example:
+  - `[fieldTags]` - This option takes an array of strings. When logging data it will look for those fields in the data values processed and then add them to the `tags` if they are.
```

Comment:
More of a general question but does Good Influx validate the `config` object?

I would like it to, but the issue is that Good reporters fail kinda silently in my experience. That said, it might be good to throw if someone sets `{ fieldTagKeys: ['myKey']}`, for instance.

Reply:
It doesn't validate to my knowledge. I see where you are coming from though and it would make sense to open a future PR to ensure that the config object sent actually matches what we expect.